src/components/calcite-video/calcite-video.scss

```diff
@@ -0,0 +1,176 @@
+    -webkit-appearance: none;
+  }
+  *::-webkit-media-controls-panel {
+    display: none !important;
+    -webkit-appearance: none;
+  }
+}
+
+//focus
+:host {
+  @include focus-style-base();
+}
+:host(:focus) {
+  @include focus-style-outset();
+}
+// show controls on hover
+:host([show-controls-on-hover]) {
+  & .calcite-video-footer {
+    visibility: hidden;
+    position: absolute;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    opacity: 0;
+    transition: all 0.15s ease-in-out;
+    background-color: var(--calcite-ui-foreground-1);
+    box-sizing: border-box;
+    z-index: 9;
+  }
+}
+:host([show-controls-on-hover]:focus),
+:host([show-controls-on-hover]:focus-within),
+:host([show-controls-on-hover]:hover) {
+  .calcite-video-footer {
+    visibility: visible;
+    opacity: 1;
+  }
+}
+
+.calcite-video-wrapper {
+  display: flex;
+  justify-content: center;
+  position: relative;
+  box-sizing: border-box;
+  height: 100%;
+  border: 1px solid var(--calcite-ui-border-3);
+  border-bottom: 0;
+  &.calcite-video-fullscreen {
+    height: 100%;
+    border-color: transparent;
+    & video {
+      height: 100vh;
+      width: auto;
+    }
+  }
+}
+
+.calcite-video-footer {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  border: 1px solid var(--calcite-ui-border-3);
+  border-top: none;
+  box-sizing: border-box;
+}
+
+.calcite-video-controls {
+  display: flex;
+  flex-direction: row;
+}
+
+.calcite-video-scrubber-wrapper {
+  position: relative;
+  padding: 0 10px;
+  border-top: 1px solid var(--calcite-ui-border-3);
+  border-bottom: 1px solid var(--calcite-ui-border-3);
+}
+
+calcite-slider.calcite-video-scrubber {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 10px 0;
+}
+.calcite-video-control-item {
+  display: flex;
+  flex-direction: row;
+  cursor: pointer;
+  border-right: 1px solid var(--calcite-ui-border-3);
+
+  &:hover,
+  &:focus {
+    background-color: var(--calcite-ui-foreground-2);
+  }
+  &:active {
+    background-color: var(--calcite-ui-foreground-3);
+  }
+}
+
+.fullscreen-control-item {
+  margin-left: auto;
+  border-right: 0;
+  border-left: 1px solid var(--calcite-ui-border-3);
+  order: 4;
+}
+
+.volume-control-item {
+  order: 2;
+}
+
+.calcite-video-time {
+  align-self: center;
+  font-size: 12px;
+  padding: 8px 12px;
+  order: 3;
+  & span:first-child {
+    font-weight: 500;
+  }
+}
+.calcite-video-control-item.volume-control-item calcite-slider {
+  width: 0;
+  opacity: 0;
+  visibility: hidden;
+  pointer-events: none;
+  align-self: flex-end;
+  transition: all ease-in-out 0.15s;
+}
+
+.calcite-video-control-item.volume-control-item:active ~ calcite-slider,
+.calcite-video-control-item.volume-control-item *:focus-within ~ calcite-slider,
+.calcite-video-control-item.volume-control-item calcite-slider:focus,
+.calcite-video-control-item.volume-control-item calcite-slider:focus-within,
+.calcite-video-control-item.volume-control-item:hover ~ calcite-slider {
```

Comment:
this seems pretty wacky, it may be better to wire up focus event which sets state via javascript and then use a single modifier class. Theoretically you would need to do some handling for screen readers in this situation as well...